### ISSUE TITLE
Oxidation states inconsistent with Wikipedia

### ISSUE THREAD

**Issue by mhfan** [lmmentel/mendeleev#199]
There are some inconsistencies between the oxidation states data and those on [Wikipedia](https://en.wikipedia.org/wiki/Oxidation_state). Is it because they are not updated synchronously?

**Comment by lmmentel** [comment]
Could you provide examples on which oxidation states do not agree?

**Comment by mhfan** [comment]
Some differences:

* Li(3) does not have a 0 extend oxidation state;
* C(6) 's main oxidation state are -4 and +4, not include -3 ~ +3 which are extend states;
* F(11) has only -1 as main oxidation state, no extended 0 and +1;
* ...

```
>>> from mendeleev.fetch import fetch_table
>>> df = fetch_table('oxidationstates')
>>> print(df.to_string())
      id  atomic_number  oxidation_state  category
0      1              1               -1      main
1      2              1                1      main
2      3              3                0  extended
3      4              3                1      main
4      5              4                0  extended
5      6              4                1  extended
6      7              4                2      main
7      8              5               -5  extended
8      9              5               -1  extended
9     10              5                0  extended
10    11              5                1  extended
11    12              5                2  extended
12    13              5                3      main
13    14              6               -4      main
14    15              6               -3      main
15    16              6               -2      main
16    17              6               -1      main
17    18              6                0      main
17    18              6                0      main
18    19              6                1      main
19    20              6                2      main
20    21              6                3      main
21    22              6                4      main
22    23              7               -3      main
23    24              7               -2  extended
24    25              7               -1  extended
25    26              7                0  extended
26    27              7                1  extended
27    28              7                2  extended
28    29              7                3      main
29    30              7                4  extended
30    31              7                5      main
31    32              8               -2      main
32    33              8               -1  extended
33    34              8                0  extended
34    35              8                1  extended
35    36              8                2  extended
36    37              9               -1      main
37    38              9                0  extended
38    39             11               -1  extended
39    40             11                0  extended
40    41             11                1      main
```

<img width="934" alt="image" src="https://github.com/user-attachments/assets/d83294b8-3eb1-4cdc-8e1e-7ce4c05c603f">

**Comment by lmmentel** [comment]
Thanks, you are correct, there are some discrepancies. I re-parsed the data from wikipedia and created a fix over at #202. Are you able to have a look and help validate?
